tests/h/traversal/roots_test.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+import mock
+
+from h.models import AuthClient
+from h.services.user import UserService
+from h.traversal.roots import AnnotationRoot
+from h.traversal.roots import AuthClientRoot
+from h.traversal.roots import OrganizationRoot
+from h.traversal.roots import OrganizationLogoRoot
+from h.traversal.roots import GroupRoot
+from h.traversal.roots import UserRoot
+from h.traversal.contexts import AnnotationContext
+
+
+@pytest.mark.usefixtures('group_service', 'links_service')
+class TestAnnotationRoot(object):
+    def test_get_item_fetches_annotation(self, pyramid_request, storage):
+        factory = AnnotationRoot(pyramid_request)
+
+        factory['123']
+        storage.fetch_annotation.assert_called_once_with(pyramid_request.db, '123')
+
+    def test_get_item_returns_annotation_resource(self, pyramid_request, storage):
+        factory = AnnotationRoot(pyramid_request)
+        storage.fetch_annotation.return_value = mock.Mock()
+
+        resource = factory['123']
+        assert isinstance(resource, AnnotationContext)
+
+    def test_get_item_resource_has_right_annotation(self, pyramid_request, storage):
```

Comment:
I'm not clear on what this test is testing? It sets the `return_value` on the `storage.fetch_annotation` mock...so I'm not sure how it is testing that it is the "right" annotation.